artifacts/mysql/mysql_db_import.yml

```diff
@@ -1,4 +1,18 @@
 ---
-- tasks:
-  - name: Import the DB 
-    mysql_db: name={{db_name}} state=import target={{db_data}} login_user={{db_user}} login_password={{db_password}}
\ No newline at end of file
+- hosts: localhost
+  connection: local
+  tasks:
+  - name: Dump DB content in a temp file (base64 encoded)
```

Comment:
@MatMaul If the DB content is not in encoded in base64 will this recipe work properly?